examples/IRServer/IRServer.ino

```diff
@@ -65,7 +65,9 @@ IRsend irsend(kIrLed);  // Set the GPIO to be used to sending the message.
 void handleRoot() {
   server.send(200, "text/html",
               "<html>" \
-                "<head><title>" HOSTNAME " Demo</title></head>" \
+                "<head><title>" HOSTNAME " Demo </title>"\
+                "<meta http-equiv=\"Content-Type\"content=\"text/html;charset=utf-8\"> \
+                </head>" \
```

Comment:
Linter errors:
```
examples/IRServer/IRServer.ino:69:  Multi-line string ("...") found.  This lint script doesn't do well with such strings, and may give bogus warnings.  Use C++11 raw strings or concatenation instead.  [readability/multiline_string] [5]
examples/IRServer/IRServer.ino:69:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
examples/IRServer/IRServer.ino:69:  Missing spaces around =  [whitespace/operators] [4]
examples/IRServer/IRServer.ino:69:  Missing spaces around <  [whitespace/operators] [3]
examples/IRServer/IRServer.ino:69:  Missing space after ;  [whitespace/semicolon] [3]
examples/IRServer/IRServer.ino:70:  Multi-line string ("...") found.  This lint script doesn't do well with such strings, and may give bogus warnings.  Use C++11 raw strings or concatenation instead.  [readability/multiline_string] [5]
```

See: https://travis-ci.org/github/crankyoldgit/IRremoteESP8266/jobs/676284478#L639-L644

Reply:
Ditto the space between http-equiv and content per earlier comment.